app/views/pregnancies/_notes.html.erb

```diff
@@ -1,8 +1,14 @@
 <section id="notes" class="notes tab-pane">
 	<div class="margin-bottom col-md-12">
 		<h1 class="border-bottom title">Notes</h1>
-    <div class="info-form-left col-md-6" id="notes_content">
-      Something here.
-    </div> 
+        <div class="info-form-left col-md-6">
+					<%= form_for [@pregnancy, @pregnancy.notes.new] do |f| %>
```

Comment:
To encourage maximum laziness in writing forms, we've been using this gem: https://github.com/bootstrap-ruby/rails-bootstrap-forms I normally try to avoid stuff like this because of the learning curve but this one's just too good to pass up. 

For consistency, could you do me a solid and replace `form_for` with `bootstrap_form_for` and the label/text area with just `f.text_field`? See the top of the edit form and the patient dashboard partial for an example of each of these.